cli/depext.ml

```diff
@@ -23,6 +23,7 @@ let run (`Root root) (`Lockfile explicit_lockfile) dry_run (`Yes yes) () =
   let depexts = D.Lockfile.depexts lockfile in
   OpamGlobalState.with_ `Lock_none (fun global_state ->
       let env = OpamPackageVar.resolve_global global_state in
+      let config = OpamFile.Config.with_depext false global_state.config in
```

Comment:
In #380 the config was only `global_state.config`, so it may be better to stick with that? In fact, all my modifications are already present in #380 so I think the opam bound can be lowered to 2.2 :)

Reply:
I think you're right. Looking at the called code nothing uses the `depext` field of the config anyway, so I'll go ahead and just use the base config.

And you're right about the bound!

Reply:
done in [ad80a13](https://github.com/tarides/opam-monorepo/pull/408/commits/ad80a134c0520e5f1e1a9fc10cf876696ce772db)